manageiq-operator/README.md

```diff
@@ -2,68 +2,128 @@
+```
 
-In project root directory, run
+### Step 2. Set up RBAC
 
 ```bash 
-$ operator-sdk run --local --namespace=<yournamespace>
+$ oc create -f deploy/role.yaml
+$ oc create -f deploy/role_binding.yaml
+$ oc create -f deploy/service_account.yaml
 ```
 
-### Option B: Run Inside Cluster
+### Step 3. Run The Operator
 
-1. Build the operator image:
+There are three different ways the operator can be run.
 
-```bash
-$ operator-sdk build docker.io/example/manageiq-operator:latest
-```
 
-2. Update the operator deployment with the new image:
++ #### Option 1: Run the latest ManageIQ Operator image from the registry in the cluster
 
-```bash
-$ sed -i 's|docker.io/manageiq/manageiq-operator:latest|docker.io/example/manageiq-operator:latest|g' deploy/operator.yaml
-```
+  The default in the operator.yaml is for the latest manageiq-operator image.
+  So no change is required. Simply create the operator.
+
+  ```bash
+  $ oc create -f deploy/operator.yaml
+  ```
 
-3. Push the new image to the registry:
++ #### Option 2: Run your own custom ManageIQ image inside the Cluster
+
+  1 - Build your operator image:
+
+    ```bash
+    $ operator-sdk build docker.io/<your_username_or_organization>/manageiq-operator:latest
+    ```
+
+  2 - Push your new custom image to the registry:
+
+    ```bash
+    $ docker push docker.io/<your_username_or_organization>/manageiq-operator:latest
+    ```
+    
+  3 - Update the operator deployment yaml file with your custom image:
+
+    ```bash
+    $ sed -i 's|docker.io/manageiq/manageiq-operator:latest|docker.io/<your_username_or_organization>/manageiq-operator:latest|g' deploy/operator.yaml
+    ```
+
+  4 - Run your custom image from the registry:
+
+    ```bash
+    $ oc create -f deploy/operator.yaml
+    ```
+
++ #### Option 3: Run locally (on your local laptop/computer, outside of the cluster)
+
+  ```bash
+  $ operator-sdk run --local --namespace=<your namespace>
+  ```
+
+### Step 4. Perform any optional custom configurations
+
+*see the OpenID-Connect example below*
+
+### Step 5. Run ManageIQ by creating the Custom Resource (CR)
 
 ```bash
-$ docker push docker.io/example/manageiq-operator:latest
+$ oc create -f deploy/crds/manageiq.org_v1alpha1_manageiq_cr.yaml
 ```
 
-### Setup RBAC and deploy the operator
+# Further Notes:
 
-```bash 
-$ oc create -f deploy/role.yaml
-$ oc create -f deploy/role_binding.yaml
-$ oc create -f deploy/service_account.yaml
-$ oc create -f deploy/operator.yaml
-```
+## Customizing the installation
+
+### Creating custom TLS secret
 
-### Create the CR to deploy ManageIQ
+ManageIQ can be run with an external Postgres or messaging server.  To do this, create the required OpenShift secret(s) with the correct parameters using the template(s) for [Postgres](templates/app/postgresql-secrets.yaml) or [messaging](/templates/app/kafka-secrets.yaml) and provide those secret names as `databaseSecret` and/or `kafkaSecret` in `manageiq.org_v1alpha1_manageiq_cr.yaml`.
```

Comment:
This should not be under `Creating custom TLS secret`.  We may even want to split the postgres and kafka instructions.